pkg/generator/keytoolimportcert_test.go

```diff
@@ -0,0 +1,106 @@
+package generator
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ForgeRock/secret-agent/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestImportCert(t *testing.T) {
+	rootCAConfig := &v1alpha1.KeyConfig{
```

Comment:
I think at some point we might want to use a helper func for this, I think this is a bit repeated but yea. 

I also wonder if some of this can be converted to test tables.